mlflow/protos/service.proto

```diff
@@ -707,6 +723,110 @@ message ListExperiments {
   }
 }
 
+message SearchExperiments {
+  option (scalapb.message).extends = "com.databricks.rpc.RPC[$this.Response]";
+
+  // Qualifier for type of experiments to be returned.
+  // If unspecified, return only active experiments.
+  optional ViewType view_type = 1;
+
+  // Maximum number of experiments desired.
+  // Servers may select a desired default `max_results` value. All servers are
+  // guaranteed to support a `max_results` threshold of at least 1,000 but may
+  // support more. Callers of this endpoint are encouraged to pass max_results
+  // explicitly and leverage page_token to iterate through experiments.
+  optional int64 max_results = 2;
+
+  // Pagination token to go to the next page based on a previous query.
+  optional string page_token = 3;
+
+  optional string filter = 4;
+
+  repeated string order_by = 5;
```

Comment:
Can we add docstrings for these parameters?

Reply:
Sure!